tests/integration/test_core.py

```diff
@@ -0,0 +1,89 @@
+import ops.model
+import juju.application
+import pytest_operator.plugin
+
+
+async def test_build_and_deploy(ops_test: pytest_operator.plugin.OpsTest):
```

Comment:
Usually we have been using some decorators such as:

- `@pytest.mark.asyncio`
- `@pytest.mark.abort_on_fail`

https://github.com/canonical/indico-operator/blob/main/tests/integration/test_charm.py

Reply:
Fixed, thanks!